Week1/Nadege_Awah/solution_to_palindrome_exercise.js

```diff
@@ -0,0 +1,15 @@
+//defines function reverseString and gives it a parameter str
+function reverseString(str) { 
+  var palindrome = "str"
+  /*
+  str is converted to array, array is reversed and converted back to string and checked if the result is equal to str.
+  */
+  if((str.split("").reverse("").join("")) == str) {
```

Comment:
Also you should always try to do both type and value comparisons using `===`

Reply:
okay.